### PR TITLE
Gas settings & speed up/cancel UI

### PR DESCRIPTION
**Pull request by greg-schrammel**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)

- matches figma
- fix cancel/speed up modal open from context click

before / after

Screenshot 2024-04-13 at 19 10 46

Screenshot 2024-04-13 at 19 08 47

Screenshot 2024-04-13 at 19 11 10

Screenshot 2024-04-13 at 19 08 53

Screenshot 2024-04-13 at 19 10 58

Screenshot 2024-04-13 at 19 02 55

### CODE REVIEW

File: src/design-system/components/Prompt/Prompt.tsx [hunk 1]
```diff
@@ -98,9 +102,10 @@ export const Prompt = ({
                   exit={{ opacity: 0 }}
                   transition={deceleratedShort}
                   background={background ?? 'surfaceMenu'}
-                  borderRadius="16px"
+                  borderRadius={borderRadius}
```
Comment (magiziz) [inline src/design-system/components/Prompt/Prompt.tsx:105]: Can you do `16px` as default if not provided ? I see we use `<Prompt />` in a lot of places.

image

File: src/design-system/components/Prompt/Prompt.tsx [hunk 2]
```diff
@@ -98,9 +102,10 @@ export const Prompt = ({
...
                   borderColor="separatorTertiary"
                   borderWidth="1px"
+                  style={{ overflow: 'hidden' }}
```
Comment (magiziz) [inline src/design-system/components/Prompt/Prompt.tsx:108]: This doesn't affect any other places where we use `<Prompt>` right ?

**Review by magiziz — APPROVED** [review]
Looks good. Just had one question

File: src/design-system/components/Prompt/Prompt.tsx (hunk 2 above)
Comment (greg-schrammel) [reply src/design-system/components/Prompt/Prompt.tsx:108]: no sirr

**Review by estebanmino — APPROVED** [review]
code lgtm but i didn't try speed up actions

**Pull request merged**